src/features/pmicControl/npm/npm1300/documentationPmic1300.tsx

```diff
@@ -1347,8 +1347,8 @@ export const documentation = (npmDevice: Npm1300 | Npm1304): Documentation => {
                     title: 'Discharge cut-off',
                     content: [
                         <p key="p1">
-                            The lowest battery operating voltage. Set this
-                            according to the battery specification.
+                            The battery capacity in mAh. Set this according to
+                            the rated capacity of the battery used.
```

Comment:
Now it looks like you copied the description of the Capacity tooltip (line 1361) into the Discharge cut-off tooltip?
Shouldn't this be:
```suggestion
                            The battery discharge cut-off voltage. Set this according to
                            the rated capacity of the battery used.
```